core-bundle/src/Resources/contao/library/Contao/StringUtil.php

```diff
@@ -777,21 +733,14 @@ public static function resolveFlaggedUrl(&$url)
 	/**
 	 * Convert the character encoding
 	 *
-	 * @param string $str  The input string
+	 * @param mixed  $str  The input string
 	 * @param string $to   The target character set
 	 * @param string $from An optional source character set
 	 *
 	 * @return string The converted string
 	 */
-	public static function convertEncoding($str, $to, $from=null)
+	public static function convertEncoding(mixed $str, string $to, string|null $from=null): string
```

Comment:
```suggestion
	public static function convertEncoding(string $str, string $to, string|null $from=null): string
```

`mixed` cannot be right if we want a stringable argument, can it?

Reply:
No we have tests that explicitly test this. So apparently it's a supported use case and the phpdoc was just wrong.

Reply:
@ausi /cc

Reply:
```suggestion
	public static function convertEncoding(bool|float|int|string|\Stringable|null $str, string $to, string|null $from=null): string
```

IMO we only want to allow stringable objects, scalars and null.

Reply:
@Toflar https://github.com/Toflar/contao/pull/12